packages/solarflare-theme/README.md

```diff
@@ -0,0 +1,3 @@
+# Solarflare Theme README
+
+TBD
```

Comment:
Would you be able to add some more details? Not blocking for merge though, that can be a follow-up